src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from "react-router-dom";
+import { Nav, Footer } from "../src/Components/index"
+import { Home, MockbeeHome, MockmanComponent} from "../src/pages/index"
```

Comment:
Instead of writing as `../src/pages/index` you can simply write it as `../src/pages`
Similarly for Line number 2

Reply:
ohh thanks for pointing this!